From 66cd349a1e5d0a33bcf029d7ad3b7727c8b10479 Mon Sep 17 00:00:00 2001
From: Gabriel Somlo <gsomlo@gmail.com>
Date: Wed, 29 Jan 2025 12:10:36 -0500
Subject: [PATCH] fedora test submodule patch

Fedora-specific patch:
Remove reference to tests submodule, so we can continue avoiding
reliance on bringing in git submodules.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f14d2cd1..f05cb0fd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -239,7 +239,7 @@ add_subdirectory(frontend)
 add_subdirectory(json)
 add_subdirectory(rust)
 
-add_subdirectory(tests/gui)
+# add_subdirectory(tests/gui)
 
 add_custom_target(nextpnr-all-bba)
 
-- 
2.48.1

